app/models/campaign.rb

```diff
@@ -24,4 +25,44 @@ def self.current
   def show_in_sidebar?
     sidebar_enabled? && sidebar_image.present?
   end
+
+  # @return [Integer] The total number of articles in the campaign.
+  delegate :count, to: :the_scope
+
+  # Get the "plucked" attribute information for the campaign's
+  # articles.
+  #
+  # @param limit [Integer] The limit of the number of articles to
+  #        fetch and render.
+  # @param attributes [Array<Symbol>] The named attributes to pluck
+  #        from the Article result set.
+  #
+  # @return [Array<Array>] The inner array is the plucked attribute
+  #         values for the selected articles.  Which means be mindful
+  #         of the order you pass for attributes.
+  #
+  # @note The order of attributes and behavior of this method is from
+  #       past implementations.  A refactor to consider would be to
+  #       create a data structure.
+  #
+  # @see `./app/views/articles/_widget_list_item.html.erb` for the
+  #      importance of maintaining position of these parameters.
+  def plucked_article_attributes(limit: 5, attributes: %i[path title comments_count created_at])
+    the_scope.limit(limit).pluck(attributes)
+  end
+
+  private
+
+  # @note [@jeremyf] My inclination was to extract a scoping method
+  #       for this.  However, I've since consolidated the logic into a
+  #       single location, so the scope is less necessary.
+  def scope
```

Comment:
should this be the_scope (and the variable @the_scope?) - it's unclear why a public method like plucked_article_attributes (or the class level `delegate:` call) would know what `the_scope` is here?

Reply:
Wow! Rubocop did some significant reworking on this PR by automagick changes.  I'll update and address.